### DISCUSSION TITLE
Classement Guilde

### DISCUSSION THREAD

**Discussion by MF4db** [Crownicles/Crownicles#828] [Nouvelles idées]
Pourquoi pas établir un "classement guildes"? Pour savoir tout d'abord qu'elles sont les meilleures guildes (avec le nombre de pts gagnés et le niveau de la guilde) et aussi pour instorer un "rivalité" entre les guildes qui pour le moment (mon opinion) n'est pas très présent (hors salon petites annonces où il y a parfois du recrutement)

**Comment by Pietagorh** [comment]
Alors en réalité y'a de la rivalité, faut juste avoir un peu plus les servs importants x)
Mais du coup pour le classement c'est prévu
